test/e2e/logs_test.go

```diff
@@ -490,4 +490,66 @@ var _ = Describe("Podman logs", func() {
 		Expect(output[0]).To(MatchRegexp(`\x1b\[3[0-9a-z ]+\x1b\[0m`))
 		Expect(output[1]).To(MatchRegexp(`\x1b\[3[0-9a-z ]+\x1b\[0m`))
 	})
+	It("podman pod logs -l", func() {
+		SkipIfRemote("Remote can only process one container at a time")
```

Comment:
sorry for the long response time. I don't think I fully understand what you want, probably because I haven't done much with podman-remote. Is what you're looking for basically a test that doesn't have the `SkipIfRemote` line and makes sure the test fails?

Reply:
I think you can remove the skip an then just don't use the pods -l line below.

Reply:
Since this only blew up on podman pod logs -l, then it will not work on Remote, since -l is not supported.